test-app/build-tools/static-binding-generator/src/main/java/org/nativescript/staticbindinggenerator/Main.java

```diff
@@ -0,0 +1,187 @@
+package org.nativescript.staticbindinggenerator;
```

Comment:
The history for this file seems to have been lost.

Reply:
It's actually a new file, because I had to rewrite the main a bit so I can bring the `interfacename.jar` `jsparser` and `staticbindinggenerator.jar` together. Anyway, it's just an entry point, so history isn't that valuable.